vitest.config.ts

```diff
@@ -5,15 +5,15 @@ export default defineConfig({
     // migration from jest
     globals: true,
     include: ['test/**/*-test.js'],
-    coverage: {
-      provider: 'v8',
-      enabled: true,
-      thresholds: {
-        lines: 86,
-        functions: 91,
-        branches: 82,
-        statements: 86,
-      },
-    },
+    // coverage: {
```

Comment:
what happened here?

Reply:
I got annoyed by the output and forgot to re-enable it :D